synapseclient/models/mixins/table_components.py

```diff
@@ -117,6 +140,346 @@ def test_import_pandas() -> None:
         raise
 
 
+def row_labels_from_id_and_version(rows):
+    return ["_".join(map(str, row)) for row in rows]
+
+
+def row_labels_from_rows(rows: List[Row]) -> List[Row]:
+    return row_labels_from_id_and_version(
+        [
+            (
+                (row.row_id, row.version_number, row.etag)
+                if row.etag
+                else (row.row_id, row.version_number)
+            )
+            for row in rows
+        ]
+    )
+
+
+async def _query_table_csv(
```

Comment:
For this method it's got the `_` prefix, but `query_table_row_set` does not. Should these be the same? Both private or not?

Reply:
That's a good catch. I think they should both be private.

Reply:
Just fixed query_table_row_set